content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,595 @@
+
+{{< /table >}}
+
+#### Wait for package install
+
+When installing a package the `crossplane xpkg install` command doesn't wait for
+the package to download and install. View any download or installation problems 
+by inspecting the `configuration` with `kubectl describe configuration`.
+
+Use `--wait` to have the `crossplane xpkg install` command to wait for a 
+package to have the condition `HEALTHY` before continuing. The command 
+returns an error if the `wait` time expires before the package is `HEALTHY`.
+
+#### Require manual package activation
+
+Set the package to require 
+[manual activation]({{<ref "../concepts/packages#upgrade-policy" >}}), 
+preventing an automatic upgrade of a package with `--manual-activation`
+
+#### Authenticate to a private registry
+
+To authenticate to a private package registry use `--package-pull-secrets` and 
+provide a list of Kubernetes Secret objects. 
+
+{{<hint "important" >}}
+The secrets must be in the same namespace as the Crossplane pod. 
+{{< /hint >}}
+
+#### Customize the number of stored package versions
+
+By default Crossplane only stores a single inactive package in the local package
+cache. 
+
+Store more inactive copies of a package with `--revision-history-limit`. 
+
+Read more about 
+[package revisions]({{< ref "../concepts/packages#configuration-revisions" >}}) 
+in the package documentation. 
+
+### xpkg login
+
+Use `xpkg login` to authenticate to `xpkg.upbound.io`, the 
+[Upbound Marketplace](https://marketplace.upbound.io/) container registry.
+
+[Register with the Upbound Marketplace](https://accounts.upbound.io/register) 
+to push packages and create private repositories. 
+
+#### Flags
+
+{{< table "table table-sm table-striped">}}
+| Short flag   | Long flag                            | Description                    |
+| ------------ | -------------                        | ------------------------------ |
+| `-u` | `--username=<username>`    | Username to use for authentication. | 
+| `-p` | `--password=<password>`    | Password to use for authentication. | 
+| `-t` | `--token=<token string>`   | User token string to use for authentication. | 
+| `-a` | `--account=<organization>` | Specify an Upbound organization during authentication. |
+{{< /table >}}
+
+
+#### Authentication options
+
+The `crossplane xkpg login` command can use a username and password or Upbound API token.
+
+Provide a username and password with the `--username` and `--password` flags or
+set the environmental variable `UP_USER` for a username or `UP_PASSWORD` for the
+password. 
+
+Use an Upbound user token instead of a username and password with `--token` or 
+the `UP_TOKEN` environmental variable. 
+
+{{< hint "important" >}}
+The `--token` or `UP_TOKEN` environmental variables take precedence over a 
+username and password.
+{{< /hint >}}
+
+Using `-` as the input for `--password` or `--token` reads the input from stdin.  
```

Comment:
we could also consider saying what happens if you pass no arguments - you get prompted for username and password:
```
❯ crossplane xpkg login
Username: jbw976
Password:
Login successful.
```